sdc/data_partitioning/README.md

```diff
@@ -0,0 +1,34 @@
+# How get the canonical data partitioning
+To get the canonical data partitioning we provide a set of scene tags filters. The filters are located in [sdc/filters.py](https://github.com/yandex-research/shifts/blob/c4969c643b4a97d69dc1a3cfb6fb3e1af17b33bc/sdc/sdc/filters.py) file. The expected usage is as follows.
+```
+from sdc.filters import DATASETS_TO_FILTERS
+from ysdc_dataset_api.dataset import MotionPredictionDataset
+
+
+train_dataset = MotionPredictionDataset(
+    dataset_path='/path/to/dataset/dir',
```

Comment:
Should we move these paths into a variable instead of duplicating them in the example?

Reply:
Most probably, train/dev/eval data paths would point to 3 different directories. Otherwise a name collision between scene files would occur.

Reply:
Done